### ISSUE TITLE
bundled cable does not input redstone to a circuit on circuit.

### ISSUE THREAD

**Issue by patc4** [amadornes/SCM-Localization-Issues#240]
![2016-11-19_13 02 11](https://cloud.githubusercontent.com/assets/15985460/20457443/59c82adc-ae59-11e6-948f-1225e047e011.png)

in the picture, on the left the gray colored redstone is the reset to the circuit. it should have set the lights on the right to shut off. it is outputting redstone from when i copied the circuit but it is not recieving new redstone signals when i input them.

**Comment by IroidoSoner** [comment]
I am having a similar issue on the new Direwolf pack. I made a pulse former circuit and put it in another circuit. It simply won't work... I even tried running a straight wire and put that into the circuit (yes it was made properly and it still wouldn't work. I put a post on the FTB Forums, here's the link https://forum.feed-the-beast.com/threads/super-circuit-maker-bug.196911/ hopefully this is fixed soon! I love this mod!

**Comment by gDKdev** [comment]
Duplicate! Using a bundled wire for circuits in circuits never really worked. Once received a signal it will stay on, because bundled wires have no in- and output

**Comment by gDKdev** [comment]
Yes, it's just a bug. And he needs to think about how to fix it. Either he also makes in- and outputs for bundled cables, uses path tracing to detect if the signal comes from the inside or outside, or something else.

**Comment by patc4** [comment]
thanks for the answer, hope to play that version soon.
